### PR TITLE
fix #31672 newsru.co.il

### LINKED ISSUE #31672: newsru.co.il
### Issue URL (Annoyance)
[http://newsru.co.il/mideast/26mar2019/gaza_0024.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fnewsru.co.il%2Fmideast%2F26mar2019%2Fgaza_0024.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 2.1.2
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | None
Filters: | AdGuard Russian,<br>AdGuard Mobile Ads,<br>AdGuard Safari,<br>AdGuard Annoyances,<br>AdGuard Base,<br>Adblock Warning Removal List,<br>Malware Domains,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/31672

### CODE REVIEW

**Review by BlazDT — CHANGES_REQUESTED** [review]
Not directly an ad which hides it, just articles from third-party site, I would put it into Annoyances.
@Alex-302 what do you think?

Please block also:
newsru

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -427,6 +427,7 @@ goodgame.ru###adinner
 alltorrents.net###adlogo
 goodgame.ru###adouter
 my-rutracker.org###adriver-240x120
+newsru.co.il##.aux-container > div > .ad
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt]: `##.ad` is safe on this site

**Comment by Alex-302** [comment]
> Not directly an ad which hides it, just articles from third-party site, I would put it into Annoyances.
> @Alex-302 what do you think?

this is ad. Please check without ad blocker.

File: RussianFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -199,6 +199,7 @@ zhzh.com.ua#?#.content > .site-name-s > h2:contains(Новини Житомир
 zhzh.com.ua#?#.aside-block:has(.adsbygoogle)
 novorosinform.org#?#.td-ss-main-sidebar .widget_text:has(> h4 > span:contains(/Новости партнеров|Актуально/))
 rgmechanics.ru#?#div:has(> center > div[id^="advertur_"])
+newsru.co.il#?#.left-column > div:has(> div > div > a[href][target="_blank"])
```
Comment (Alex-302) [inline RussianFilter/sections/css_extended.txt]: `#?#` - remove `?`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by BlazDT — APPROVED** [review]

**Review by slavaleleka — CHANGES_REQUESTED** [review]
I guess this part of header should be unblocked.
`~newsru.co.il##.social-buttons` to SocialFilter > general_elemhide.txt

Screenshot 2019-06-27 at 12 24 01
_____________________________

and block that left-over please (for content blockers only)

Screenshot 2019-06-27 at 12 31 33

File: RussianFilter/sections/css_extended.txt [hunk 3]
```diff
@@ -199,6 +199,7 @@ zhzh.com.ua#?#.content > .site-name-s > h2:contains(Новини Житомир
 zhzh.com.ua#?#.aside-block:has(.adsbygoogle)
 novorosinform.org#?#.td-ss-main-sidebar .widget_text:has(> h4 > span:contains(/Новости партнеров|Актуально/))
 rgmechanics.ru#?#div:has(> center > div[id^="advertur_"])
+newsru.co.il##.left-column > div:has(> div > div > a[href][target="_blank"])
```
Comment (Alex-302) [inline RussianFilter/sections/css_extended.txt]: too wide selector. Make it more specific

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**